cli/demo_sidechain.sh

```diff
@@ -81,10 +81,6 @@ echo "* Issue ${INITIALFUNDS} tokens to Alice's incognito account (on worker 1)"
 ${CLIENTWORKER1} trusted set-balance ${ICGACCOUNTALICE} ${INITIALFUNDS} --mrenclave ${MRENCLAVE} --direct
 echo ""
 
-echo "* Waiting 2 seconds"
-sleep 2
-echo ""
-
```

Comment:
So the client call (i.e. listener) blocks until the sidechain block is imported (meaning state is updated)? That ensures when we continue that a getter will return the correct state?

Reply:
Yes, currently the cli waits until `InSidechainBlock` by default. Ideally, we will introduce the option to choose until which state we want to listen to, just like in the susbtrate-api-client. But that's an open issue: https://github.com/integritee-network/worker/issues/207